src/features/barn/BarnInside.tsx

```diff
@@ -68,6 +70,35 @@ export const BarnInside: React.FC = () => {
 
   const nextLevel = Math.min(level + 1, 3) as Exclude<AnimalBuildingLevel, 1>;
 
+  const {
+    x: floorX,
+    y: floorY,
+    height: floorHeight,
+    width: floorWidth,
+  } = ANIMAL_HOUSE_BOUNDS.barn[level];
+
+  // Organise the animals neatly in the barn
+  const organizedAnimals = useMemo(() => {
+    const animals = getKeys(barn.animals).map((id) => ({
+      ...barn.animals[id],
+    }));
+
+    const maxAnimalsPerRow = Math.floor(floorWidth / ANIMALS.Cow.width);
+    const verticalGap = 0.5; // Add a 0.5 grid unit gap between rows
+
+    return animals.map((animal, index) => {
+      const row = Math.floor(index / maxAnimalsPerRow);
+      const col = index % maxAnimalsPerRow;
+      return {
+        ...animal,
+        coordinates: {
+          x: col * ANIMALS.Cow.width,
```

Comment:
Should a note or build break be added somewhere to avoid Cow and Sheep dimensions ever being different?